fix(curriculum): replace self-closing with void element#55342 (#56303)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations/css-foundations-lesson-f.md

Okay, you went over quite a bit so far. The only thing left for now is to go over how to add all this CSS to your HTML. There are three methods to do so.

> External CSS is the most common method you will come across, and it involves creating a separate file for the CSS and linking it inside of an HTML’s opening and closing `<head>` tags with a void element `<link>`:

Reviewer: ```suggestion
External CSS is the most common method you will come across, and it involves creating a separate file for the CSS and linking it inside of an HTML’s opening and closing `<head>` tags with a `<link>` element (which is a void element and doesn't require a closing tag):
```

Reviewer 2: This change has not been applied.

> External CSS is the most common method you will come across, and it involves creating a separate file for the CSS and linking it inside of an HTML’s opening and closing `<head>`  tags with a `<link>` element (which is a void element and doesn't require a closing tag):

Reviewer 2: ```suggestion
External CSS is the most common method you will come across, and it involves creating a separate file for the CSS and linking it inside of an HTML’s opening and closing `<head>` tags with a `link` element. The `link` element is a void element, so should not have a closing tag.
```

> First, you add a void element `<link>` inside of the opening and closing `<head>` tags of the HTML file. The `href` attribute is the location of the CSS file, either an absolute URL or, what you’ll be utilizing, a URL relative to the location of the HTML file. In the example above, you are assuming both files are located in the same directory. The `rel` attribute is required, and it specifies the relationship between the HTML file and the linked file.

Reviewer: ```suggestion
First, you add a `<link>` tag inside of the opening and closing `<head>` tags of the HTML file. The `href` attribute is the location of the CSS file, either an absolute URL or, what you’ll be utilizing, a URL relative to the location of the HTML file. In the example above, you are assuming both files are located in the same directory. The `rel` attribute is required, and it specifies the relationship between the HTML file and the linked file.
```

Reviewer 2: This change has not been applied.
